DigitalLearningSolutions.Data.Tests/Services/TutorialContentServiceTests.cs

```diff
@@ -234,6 +234,66 @@ public void Get_tutorial_information_nextSection_skips_sections_full_of_archived
             }
         }
 
+        [Test]
+        public void Get_tutorial_information_nextSection_returns_section_with_only_diagnostic_assessment()
+        {
+            // Given
+            const int candidateId = 74411;
+            const int customisationId = 5852;
+            const int sectionId = 150;
+            const int tutorialId = 634;
+
+            const int nextSectionId = 151; // All tutorials are CustomisationTutorials.Status = 0, though some DiagStatus = 1
+
+            // When
+            var tutorial = tutorialContentService.GetTutorialInformation(candidateId, customisationId, sectionId, tutorialId);
+
+            // Then
+            tutorial.Should().NotBeNull();
+            tutorial!.NextSectionId.Should().Be(nextSectionId);
+        }
+
+        [Test]
+        public void Get_tutorial_information_nextSection_returns_section_with_only_post_learning_assessment()
+        {
+            // Given
+            const int customisationId = 10820;
+            const int candidateId = 1;
+            const int sectionId = 104;
+            const int tutorialId = 331;
+
+            const int expectedNextSectionId = 105;

```

Comment:
Similar comment to the one in the previous test would be useful I think? In general I prefer making code more clear over adding comments, but I think our data tests are one case where we could probably be more generous with comments. As many of the rely on knowing the state of the db, and knowing how rather complicated bits of the queries work together which makes it harder to understand the tests. Comments like the one you added above make that a lot easier 🙂

Reply:
Yeah absolutely, added in 08afd86 🙂